imagegw/shifter_imagegw/dockerv2.py

```diff
@@ -404,11 +410,6 @@ def do_token_auth(self, auth_loc_str, creds=False):
 
         if resp.status != 200:
             raise ValueError('Bad response getting token: %d', resp.status)
-        # The json content type can include a extra bit defining the character set
-        # so we just check the beginning.  Worst case is the json.loads will throw
-        # an error just after this.
-        if not resp.getheader('content-type').startswith('application/json'):
```

Comment:
Why was this dropped?  Does the nvcr case not return json?

Reply:
This part is from your original PR, but I can check if it's still needed.